.github/workflows/callable.test.yml

```
@@ -19,9 +19,11 @@ jobs:
         - name: Setup Gradle
           uses: gradle/actions/setup-gradle@v3
         - name: Run integration tests
-          run: ./gradlew test
+          run: ./gradlew test sonar
           env:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

yamllint

[warning] wrong indentation: expected 14 but found 12

(indentation)